app/components/table-group.hbs

```diff
@@ -0,0 +1,43 @@
+{{#if (eq @models.length 0)}}
+    <p class="empty-model">No records found</p>
+{{else}}
+    <table class="table-group">
+        <tr class="table-group-header-row">
+            {{#each @tableOptions as |object|}}
+                <th class={{if (eq @sortField object.value) "table-group-header sorted" "table-group-header"}}
+                    {{on "click" (fn @onHeaderClick object.value)}}>
+                    <p class={{if (eq @sortDirection 'ASC') "asc" "desc"}}>
```

Comment:
Can such logic be moved outside of view?

Reply:
Might be it's a chance to create variable for holding sortDirection. Something like 
`const SORT_DIRECTION = {
    ASC: 'asc',
    DESC: 'desc'
};`